Add the guide to init your first terraform infrastructure (#1072)
scaleway/terraform-provider-scaleway: docs/guides/how_to_terraform.md

# How to init your infrastructure with Terraform

> Scaleway's products are can be used and deployed through Terraform.

Reviewer: ```suggestion
Scaleway's products can be used and deployed through Terraform.
```

> The purpose of this guide is to help them deploying their first infrastructure in terraform, according some good practices.

Reviewer: ```suggestion
The purpose of this guide is to help users deploy their first infrastructure in Terraform, according to some good practices.
```

## The workflow

> In order to deploy your infrastructure properly, we would recommend to follow the following workflow, as detailled in the blog article.

Reviewer: ```suggestion
To deploy your infrastructure, we recommend you follow the workflow, as detailed in the blog article.
```
